client/src/pages/Result/Result.tsx

```diff
@@ -0,0 +1,49 @@
+import { useLocation, useNavigate } from "react-router-dom";
+import { Leaderboard } from "../../contexts/quiz.context.types";
+
+import "./Result.css";
+import { Location } from "./Result.types";
+
+const Result = () => {
+
+  const navigate = useNavigate();
+  const location:Location = useLocation();
+  
+  const score = location.state?.score;
+  const leaderboard = location.state?.leaderboard;
+  const filteredLeaderboard = leaderboard?.sort((a,b) => b.score - a.score);
```

Comment:
You can add these functions to the util file

Reply:
No, it used only 1 time. I don't think to put it in util file